bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/ProjectRepository.cs

```diff
@@ -123,6 +123,7 @@ public async Task<IEnumerable<Core.SecretsManager.Entities.Project>> GetManyById
         {
             var dbContext = GetDatabaseContext(scope);
             var projects = await dbContext.Project
+                                    .Include(p => p.Secrets)
```

Comment:
Do we want to have another method on the repository? Including secrets will be more intensive for the database than just listing projects.

Reply:
Is this only for delete? If so that's perfectly fine. I'm worried that we use this in more places?

Maybe rename the method to  `GetManyWithSecretsByIds`

Reply:
Yes, it's only used for delete- currently the only reference outside of tests is within the `DeleteProjects` function in `DeleteProjectCommand.cs`

I renamed the method 👍